example/timer/odp_timer_accuracy.c

```diff
@@ -100,10 +109,9 @@ typedef struct test_global_t {
 	uint64_t         period_tick;
 	double           period_dbl;
 	odp_fract_u64_t  base_freq;
-	odp_shm_t        log_shm;
-	test_log_t	*log;
 	FILE            *file;
 	char		 filename[MAX_FILENAME + 1];
+	test_shm_t	*shm;
 
 } test_global_t;
```

Comment:
It would be cleanest to allocate entire test_global_t from SHM. All variables would be from the same memory and on the same level on the struct. It would safer option also if running in process mode.

The move of test_global_t into SHM could be done in a separate patch to minimize size this patch.